### ISSUE TITLE
Redshift Connector wants a newer Boto3 version than spec'd by AWSWrangler

### ISSUE THREAD

**Issue by jpl-jengelke** [awslabs/aws-data-wrangler#728]
Pipenv doesn't like to install certain older versions of Boto3 and AWS Data Wrangler. It looks like this is a generic problem for anyone trying to use an older version in the range `>1.12.49` and `<1.16.8`. I didn't test it with `pip` but I suspect the same issue. 

This is reasonably self-explanatory: 
```
There are incompatible versions in the resolved dependencies:
  boto3==1.14.59 (from -r /var/folders/0_/0mssd1656q78c47k0_yk7sn00000gp/T/pipenv0jy8wrxhrequirements/pipenv-77kko2zf-constraints.txt (line 6))
  boto3<2.0.0,>=1.12.49 (from awswrangler==2.0.0->-r /var/folders/0_/0mssd1656q78c47k0_yk7sn00000gp/T/pipenv0jy8wrxhrequirements/pipenv-77kko2zf-constraints.txt (line 3))
  boto3<2.0.0,>=1.16.8 (from redshift-connector==2.0.881->awswrangler==2.0.0->-r /var/folders/0_/0mssd1656q78c47k0_yk7sn00000gp/T/pipenv0jy8wrxhrequirements/pipenv-77kko2zf-constraints.txt (line 3))
```

The oldest version of [Redshift-Connector](https://pypi.org/project/redshift-connector/#history) available is now [v2.0.384](https://github.com/aws/amazon-redshift-python-driver/blob/v2.0.384/requirements.txt).

So [AWSWrangler](https://pypi.org/project/awswrangler/) claims it supports [boto3>=1.12.49,<2.0.0](https://github.com/awslabs/aws-data-wrangler/blob/335ce5a4f89d2ba16546d44e57e013ccf1cf9772/requirements.txt#L1) but Redshift-Connector requires [boto3>=1.16.8,<2.0.0](https://github.com/aws/amazon-redshift-python-driver/blob/2493edb5a7a66bb975531d3d48ac654dce4f0940/requirements.txt#L4).  

In summary, this means that AWSWrangler should require `boto3>=1.16.8`.

**Comment by jaidisido** [comment]
Thanks for raising this, boto3 and botocore dependencies have now been upgraded. It will be available in the next version
